waiter/bin/ci/saml-idp-server-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
We should add documentation for usage and examples.

Reply:
done